packages/editor-ui/src/components/Node.vue

```diff
@@ -256,6 +355,16 @@ export default mixins(externalHooks, nodeBase, nodeHelpers, workflowHelpers).ext
 				this.$emit('duplicateNode', this.data.name);
 			});
 		},
+		getTriggerNodeTooltip (): string | undefined {
+			if (this.nodeType !== null && this.nodeType.hasOwnProperty('eventTriggerDescription')) {
+				return this.nodeType.eventTriggerDescription;
+			} else {
+				return `Waiting for you to create an event in ${this.getTrimmedWebhookNodeName()}`;
+			}
+		},
+		getTrimmedWebhookNodeName() {
+			return this.name.replace(/Trigger/g, "");
```

Comment:
we cannot use `name` here.. this is a prop that's passed from `NodeView.vue` that specifies the node name which users can edit... What we want here is the name of the nodeType so `this.nodeType.name`... also maybe this should not be global, but just at the end of the name

Reply:
I change this to this.nodeType.displayName here instead of this.nodeType.name since we need the name of node.